.github/workflows/pr_infra.yaml

```diff
@@ -5,13 +5,12 @@ on:
   pull_request:
     types:
       - opened
-      - edited
       - synchronize
       - reopened
       - ready_for_review
     paths:
       # Trigger the workflow when resources are modified
-      - "infra/resources"
+      - "infra/resources/**"
```

Comment:
**note**: Related to [this comment](https://github.com/pagopa/trial-system/pull/41#discussion_r1611439271)